ang/crmStatusPage/StatusPageCtrl.js

```diff
@@ -12,8 +12,19 @@
         apiCalls = (apiCalls || []).concat([['System', 'check', {sequential: 1}]]);
         $('#crm-status-list').block();
         crmApi(apiCalls, true)
-          .then(function(result) {
-            $scope.statuses = result[result.length - 1].values;
+          .then(function(results) {
+            $scope.statuses = results[results.length - 1].values;
+            results.forEach(function(result) {
+              if (result.is_error) {
+                var error_message = result.error_message;
+                if (typeof(result.debug_information) !== 'undefined') {
+                  error_message += '<div class="status-debug-information">' +
+                      '<b>Debug information:</b><br>' +
+                      result.debug_information + '</div>';
+                }
+                CRM.alert(error_message, 'API error', 'error');
```

Comment:
Good thinking. And if I change the signature of `refresh()` to include a title, then I can say *which* operation failed. Updated (lmk if that signature change seems a problem, note this includes a change to the "hush" actions which use `refresh()` to set a statuspref.

This change assumes that the list of actions is a single item (array of actions, string title). Is there a case where this page triggers multiple actions in a single shot?

![image](https://cloud.githubusercontent.com/assets/105608/26267108/75fbbd3c-3d3c-11e7-9691-a149fd142217.png)